build.gradle

```diff
@@ -111,3 +121,32 @@ aemCompose {
     includeProjects ':app:*'
     includeProjects ':content:*'
 }
+
+fork {
+    config {
+        copyFile()
+        moveFile([
+                "/example": "/{{projectName}}"
+        ])
+        replaceContent([
+                "com.company.aem.example": "{{package}}",
+                "example": "{{projectName}}",
+                "Example": "{{projectLabel}}"
+        ])
+        clean()
+        action({
```

Comment:
this is only an example custom action implementation to demonstrate Fork Plugin DSL / https://github.com/neva-dev/gradle-fork-plugin